drv/fpga-api/src/lib.rs

```diff
@@ -265,6 +265,15 @@ impl FpgaUserDesign {
         Ok(v)
     }
 
+    pub fn read_bytes(
```

Comment:
Now that you've got this function, you should modify `pub fn read<T>` above to use it.